Tests/Test_WD_Task.c

```diff
@@ -25,7 +25,7 @@ void Task2(void *p_arg){
         BSP_Light_Toggle(EXTRA);
         OSTimeDly(25, OS_OPT_TIME_DLY, &err);
         // Comment out the following lines to test watchdog timeout
-        OSMutexPend(&WDog_Mutex, 0, OS_OPT_PEND_BLOCKING, NULL, &err);
+        RTOS_BPS_MutexPend(&WDog_Mutex, 0, OS_OPT_PEND_BLOCKING);
```

Comment:
I think we always pass `0` as the second parameter, so you can remove it and just pass it internally